---
title: "GFP Gene Expression and EGFP Protein Purification"
layout: post
categories: Research
---

![GfpProjectCover](/assets/img/research/gfp/cover.png){: width="75%" }

This experiment aims to transform E. coli (DH5α) cells to express the GFP gene using ligation product between pET28c expression vectors with GFP DNA (obtained from pET23-gfp plasmid). In addition, this experiment also aims to purify, analyse, and verify the expression of EGFP protein in the E. coli strain, BL21(DE3) cell.


Green Fluorescent Protein (GFP) is a 27kDa protein consisting of 238 amino acids found naturally in Aequorea Victoria Jellyfish. GFP is able to fluoresce under UV light, this ability is conserved by its compact 3D structure shielded in the centre of an 11 B-sheets barrel structure, its chromophore. Wild-type GFP can be engineered to enhance its expression in mammalian cells as EGFP where its chromophore contains F64L and S65T mutations, resulting in better chromophore folding to give stronger fluorescence. This suggests improved sensitivity upon expressions in mammalian cells compared to GFP. 

The pET23-gfp plasmid was digested with NdeI and HindIII and analysed under UV light using agarose gel electrophoresis.

## Analysis of Digested and Undigested pET23-gfp Plasmid

Prior to PCR reaction, the quality of digested pET23-gfp plasmid was analysed. Analysis showed that unrestricted plasmid DNA exists in two conformations: supercoiled DNA and an open-circular DNA. With the same fragment size and due to its more compact structure, supercoiled DNA migrates through the gel at a faster rate than an open-circular DNA. Meanwhile, aopen-circular DNA contains single-stranded breaks in its circular conformation, introducing nicks which permits phosphodiester bond rotation. This leads to the release of its supercoiled formation. Hence, forming two bands in undigested pET23-gfp DNA. On the other hand, digested pET23-gfp DNA using restriction enzymes linearised plasmid DNA, introducing more friction as it migrated through the gel than supercoiled DNA but less friction than open-circular DNA. This explains the presence of a band in between supercoiled and open-circular DNA.

![Figure1Pet23](/assets/img/research/gfp/Analysis%20of%20digested%20pET23-gfp%20plasmid.png){: width="60%" }
*Figure 1. Agarose gel electrophoresis of cut and uncut pET23-gfp plasmid.*
<div style="margin-top:-10px;"></div>
*DNA samples were loaded to 1.2% (w/v) agarose gel (stained with Gel Red solution). Gel Red intercalates between bases and illuminates under UV light. Lane M shows 1 kB DNA ladder with its known DNA fragment size (kb) provided on the left-hand side of the gel photograph. ‘Uncut’ lane denotes undigested pET23-gfp plasmid. ‘Cut’ lane denotes digested pET23-gfp plasmid. Orange arrows symbolise the two DNA fragments of uncut pET23-gfp plasmid. Meanwhile, the yellow arrow indicates a DNA fragment of digested pET23-gfp plasmid.*

## Analysis of Polymerase Chain Reaction (PCR) Product

Ligation of GFP DNA into expression vector pET28c was performed prior to transforming E. coli competent cell, DH5a. PCR was then set up to amplify the amount of DNA obtained. The PCR product was then analysed in agarose gel electrophoresis.

![Figure2Pcr](/assets/img/research/gfp/Analysis%20of%20PCR%20product.png){: width="70%" }
*Figure 2. Agarose gel electrophoresis analysis of PCR Product.*
<div style="margin-top:-10px;"></div>
*The DNA sample was loaded to the 1.2% (w/v) agarose gel (stained with Gel Red solution). Gel Red intercalates between bases and illuminates under UV light. Lane M shows a 100 bp DNA ladder alongside its known mass (ng) and fragment size (bp) for each band included on the left-hand side. PCR samples were loaded to lanes 1-5. Colony samples were added to Lane 1 (Sample Colony #1) and Lane 2 (Sample Colony #2). Controls were added to lane 3 (undigested pET28c expression vector with no GFP insert, obtained from positive colony LBK plate), lane 4 (Positive Control containing pET23-gfp DNA) and Lane 5 (water control). Pink arrows indicate GFP DNA in transformed colonies. The light blue arrow indicates GFP DNA from the pET23-gfp plasmid. The region of PCR samples was highlighted by using T7 primers (T7 promoter & T7 terminator).*

## EGFP Protein Purification Quality Control using Western Blot

Once verified, His-tagged EGFP protein was made by transforming BL21(DE3) cells and purified using Ni-NTA chromatography and analysed by SDS-PAGE and Western Blot.

![Figure3Egfp](/assets/img/research/gfp/Western%20Blot%20of%20EGFP%20Purification.png){: width="80%" }
*Figure 3. Analysis of EGFP Purification using Western Blot.*
<div style="margin-top:-10px;"></div>
*Lane M represents a protein marker with its known fragment size denoted on its left side. Lane ‘Un’ represents a lane with an uninduced sample. Lane ‘Induc’ represents a lane with an induced sample. Lane ‘Insol’ represents a lane loaded with insoluble fraction samples. Lane ‘Sol’ represents the lane with a soluble fraction sample. Lane ‘FT’ represents the lane loaded with flow-through samples. Lanes ‘W1’ and ‘W2’ symbolize lanes loaded with Washes 1 and Washes 2 respectively. Lane ‘E1’ and ‘E2’ represent lanes loaded with Elution 1 and Elution 2 respectively. BL21(DE3) cell pellet was treated with DNAsel, then centrifuged to obtain soluble fraction sample (supernatant) and insoluble fraction sample (pellet). FT sample was collected from supernatant fluid from centrifugation after his-tagged EGFP bound to the resin & before any wash buffers were added. Samples W1 and W2 were collected upon resin washes with a wash buffer (0.5 mM NaCl, 60 mM imidazole, 20 mM Tris.HCl, pH 7.9). Samples E1 and E2 were collected upon the addition of an elution buffer (1 M imidazole, 0.5 M NaCl, 20 mM Tris.HCl, pH 7.9). The Western Blot was probed with a HisprobeTM-HRP working solution, developed with DiaminoBenzidine (DAB) for chromogenic detection.*
