---
title: "Investigating HIV-1 Nef Hijacking of Cell Machinery for Pathogenesis"
layout: post
categories: Research
---

![HivProjectCover](/assets/img/research/hiv/cover.png){: width="70%" }

This research aims to understand the function of Nef in HIV-1 infection by characterizing its role in CD4 downregulation and interactions with Hck-SH3 protein, employing biophysical approaches. Despite lacking enzymatic activity and a clear mechanism of action, Nef significantly enhances viral replication and pathogenesis, underscoring its importance in HIV-1 pathogenesis and potential as a target for therapeutic intervention.


The Human Immunodeficiency Virus 1 (HIV-1) is a life-threatening retrovirus which interferes with and attacks the immune system of the body. HIV-1 exhibits high transmissibility, a higher risk, and is found as the causative agent of developing Acquired immune deficiency syndrome (AIDS). Currently, no cure or vaccines are available for prevention. The HIV-1 genome consists of 9 open reading frames encoding 15 different mature proteins, which are all involved in hijacking cells of the immune system. Among these proteins, Nef is a pathogenic factor which was found not required for HIV-1 infection but significantly increases viral replication and pathogenesis. Regardless of treatment, untreated patients with Nef-deficient strains of HIV-1 are less likely to develop AIDS, proving the significant effects the proteins must have. The full extent of Nef’s potential has not been recognised due to its lack of enzymatic activity and its poorly understood mechanism of action. Hence, this research aimed to characterise the function of Nef with respect to CD4 downregulation and interactions with Hck-SH3 protein using the biophysical approaches described below.

To carry out this research, two mutations were introduced to the Nef protein.

<table>
  <caption style="font-style: italic; ">Table 1. List of Nef mutants used in this research project.</caption>
<thead>
  <tr><th>No</th><th>Abbreviations used</th>
<th>Mutated residues</th> <th>Description</th>
</tr>
</thead>
<tbody>
  <tr><td>1</td> <td>WT</td> <td>-</td> <td>Wildtype HIV-1 Nef protein</td></tr>
<tr><td>2</td> <td>G2A Nef</td> <td>Glycine to Alanine at position 2</td> <td>Mutation at the G2 myristoylation site</td></tr>
<tr><td>3</td> <td>AxxA Nef</td> <td>Proline to Alanine at positions 76
and 79</td>
<td>Mutation at the hypothesised SH3
domain binding interface</td></tr>
</tbody>
</table>

## HIV-1 Evades Immune Response by Nef-Induced CD4 Downregulation

This series of experiments is designed to examine the impact of the Nef protein on CD4 downregulation in HeLa cells.

![Figure1ConfocalMicroscopy+FlowCytometry](/assets/img/research/hiv/Confocal%20Microscopy%20+%20Flow%20Cytometry.png){: width="75%" }
*Figure 1. CD4 expression in HeLa cells in the presence of WT-Nef, and Nef mutants (G2A and AxxA).*
<div style="margin-top:-10px;"></div>
*(A) Wild-type HIV-Nef IRES GFP. (B) Mutant G2A HIV-Nef IRES GFP. (C) Mutant AxxA
HIV-Nef IRES GFP. (D) Negative control. Immunofluorescence was used to co-localize Nef
in HeLa cells, visualized using confocal microscopy. HeLa cells' nucleus was stained (blue)
with DAPI (4',6-diamidino-2-phenylindole), Nef-GFP transfected cells (green), and anti-Nef
antibody (red). The Image was further analyzed on an image processor, FIJI.
<br>(E) Flow cytometry measuring mean fluorescence intensity (MFI) of CD4 in HeLa cells that
express wildtype Nef (WT), two Nef mutants (G2A and AxxA), cells not expressing Nef or
CD4 (-ve control), and cells not expressing Nef, but with normal CD4 levels (+ve control).
All data are normalised to a scale of 0-100. Sample size = 10, with standard deviation
showing +/- 1s.d. ANOVA testing p<0.001.*

To investigate the effects of mutations on Nef localisation, we performed
immunofluorescence using confocal microscopy. Figure 2A shows WT Nef localised
predominantly at the cell surface. In contrast, Figure 2B shows that the G2A mutation
disrupts this localisation, resulting in a diffuse cytoplasmic distribution with reduced cell
surface presence. Figure 2C indicates that the AxxA mutation has a milder effect, with Nef
AxxA still localising at the cell surface and partially in the cytoplasm. Figure 2D displays the
negative control (no Nef expression). These findings suggest that the G2A mutation most significantly disrupts Nef localisation,
shifting it from the membrane to the cytoplasm.

To assess the functional consequences of these mutations, we quantified CD4 levels in cells
expressing Nef variants using flow cytometry (Figure 2E). Compared to the positive control,
cells expressing WT Nef and AxxA show significantly reduced MFI, indicating effective
CD4 downregulation. Notably, AxxA-expressing cells exhibit even lower MFI than WT,
suggesting enhanced activity. In contrast, G2A-expressing cells show only a slight reduction
in CD4 compared to the control, confirming impaired function.

Together, the microscopy and flow cytometry results show that the G2A mutation disrupts
Nef’s ability to localise to the membrane and downregulate CD4 through lysosomal
degradation. This highlights the importance of glycine at position 2 in helping Nef hijack
endocytic pathways for immune evasion.

## Investigating the Effects of G2A and AxxA Mutations in the Secondary Structure of Nef Protein

Circular Dichroism was used as protein quality control, ensuring that mutations introduced in
Nef were able to generate proteins that were able to fold properly into the 3D structure,
similar to the wild-type construct with small structural differences.

![Figure2Circular0Dichroism](/assets/img/research/hiv/Circular%20Dichroism.png){: width="100%" }
*Figure 2. Analysis on the integrity of the secondary structure of wild-type and mutant Nef proteins.*
<div style="margin-top:-10px;"></div>
*(A) Circular Dichroism characterisation of Nef variants. (B-D) K2D3 software analysis comparing input and predicted spectra. Purple shows the predicted spectra, green shows input spectra. Standard deviation was used as Error bars (B) WT Nef (C) G2A Nef (D) AxxA Nef. (E) K2D3 software predictions of secondary structure.*

The molar extinction values for both WT and AxxA Nef proteins were comparable (Figure
2A), whereas the G2A mutant exhibited a slightly reduced value, suggesting a slight
decrease in alpha-helical content. This observation is supported by K2D3 structural
predictions (Figures 2B-E), which also indicate a minor reduction of less than 1% decrease
in alpha-helical structure in the G2A variant. However, these differences were minimal and
not statistically significant. Overall, circular dichroism analysis suggests that the G2A and
AxxA mutations do not substantially alter the secondary structure of Nef. Therefore, with the
structural integrity largely preserved, further biophysical characterisation of Nef constructs
can be carried out.

## Biophysical Characterisation of Nef Protein and Hck-SH3 Protein

To better understand the influence of Nef-SH3 interaction in HIV-1 viral
growth, several biophysical assays were employed to analyse the binding ability of both Nef mutants
to SH3 protein using wild-type Nef as a control.

<div markdown="1" style="padding-left:30px;">

### Utilising an Optical Technique to Monitor Molecular Interactions in Real-Time

This experiment investigates the binding abilities of G2A and AxxA Nef mutants to GST-tagged SH3 protein using wild-type Nef as a positive
control analysed using its dissociation constant (Kd) by Surface Plasmon Resonance (SPR). This optical biosensing experiment relies on the
ligand-analyte binding causing a change in the refractive light index detected on a sensorgram. To do this, GST-tagged SH3 protein was immobilised
using an amine-coupling reaction on an SPR sensor chip while its analyte, Nef proteins (wild-type and mutants) flowed on the surface at different
lanes in three different concentrations.

![Figure3SPR](/assets/img/research/hiv/SPR.png){: width="70%" }
*Figure 3. SPR binding assay on interactions between Nef to GST-tagged SH3 domain.*
<div style="margin-top:-10px;"></div>
*(A) Binding assay was carried out for WT Nef at 40 nM, 200 nM and 1000 nM. (B) Binding assay for G2A Nef at 40 nM, 200 nM and 1000 nM. (C) Binding assay for AxxA Nef at 40 nM, 200 nM and 1000 nM. (D) Nef Dissociation constant for WT Nef, G2A Nef, and AxxA Nef were measured.*

Surface Plasmon Resonance (SPR) was used to assess the binding affinity of WT, G2A, and
AxxA Nef variants to GST-tagged SH3 protein at three concentrations: 40 nM, 200 nM, and
1000 nM. SH3 (ligand) was immobilised on a sensor chip via amine-coupling, and Nef
variants (analytes) were flowed over the surface. Binding events were detected as changes in
the refractive index, shown in the respective sensorgrams (Figures 3A-C).

Across all concentrations, WT Nef showed the strongest binding affinity to SH3. In contrast,
the AxxA mutant showed no detectable binding and no dissociation constant (Figure 4D),
suggesting that mutating Proline-76 and Proline-79 to Alanine in its PxxP polyproline motif
abolishes SH3 binding. Although sensorgram spikes were observed, they are likely artefacts
from buffer mismatch.

The G2A mutant retained some binding to SH3, though at a reduced level compared to WT,
indicating that while Gly2 is important, its mutation partially impairs Nef-SH3 interaction,
unlike the complete disruption seen with the AxxA mutant.

### Quantitative Analysis of Nef-SH3 Interactions to Deduce the Specific Nef-SH3 Binding Interface

To investigate the Nef-SH3 interaction at the molecular level, Nuclear Magnetic Resonance
(NMR) spectroscopy was utilised to compare the chemical environment of apo Nef and
SH3-bound Nef. Using a non-destructive approach, 15N-labelled WT Nef was analysed with
and without the presence of unlabelled SH3 protein. Chemical shift perturbation (CSP)
values were used to identify residue-level changes upon complex formation, with CSP > 0.06
indicating significant interaction (Figure 4A).

![Figure4NMR](/assets/img/research/hiv/NMR.png){: width="72%" }
*Figure 4. Analysis on the binding interface between Nef-SH3 interaction using NMR approach.*
<div style="margin-top:-10px;"></div>
*(A) Graph showing the chemical shift perturbation values. Any points above the pink threshold (CSP values >0.06) indicate a significant change in chemical shift and are likely residues that are affected by binding of Nef to SH3, Hck-SH3 domain is highlighted in pink. (B) 3D-molecular structure of HIV-Nef protein core. Significant changes upon Nef binding to SH3 cyan, SH3. Image visualized using PyMOL Software, PDB accession code: 6B72.*

Results show that residues within helices α1 and α2 exhibit marked CSPs upon SH3 binding,
suggesting structural rearrangement in this region. Notably, once mapped in PyMOL, these
residues exhibit proximity to Nef's conserved PxxP motifs identified from SPR analysis:
residues Proline-76 and Proline-79, which are critical for SH3 binding (Figure 4B).


</div>

Taken altogether, the series
of biophysical technique analyses suggest that Proline-76 and Proline-79 of the Nef genome are
essential for Nef to interact with Hck-SH3 protein. In addition, hydrophobic residues in proximity to
Proline-76 and Proline-79 serve as the binding region for Nef-SH3 complex formation.

## Conclusion

The findings obtained from this project suggest that both G2A and
AxxA Nef mutants do not have significant changes in their overall protein structure. This highlights
that the interaction between Nef and the SH3 domain and its effects in downregulating CD4 are
residue-specific, rather than due to global structural changes. Therefore, it is concluded that G2 of
Nef is crucial for its downregulation of CD4, as it is the site of myristoylation and therefore
responsible for membrane anchorage, which allows Nef to hijack the clathrin AP2-dependent
endocytosis pathway to downregulate CD4. In addition, it can also be deduced that Nef selectively
activates Src family kinase pathways (such as Hck in macrophages). This requires the Nef-mediated
activation of Hck via the Hck-SH3 domain where this project showed that both PxxP polyproline
type II helix and the hydrophobic residues within the core of Nef interact with the SH3 RT-loop to
provide high affinity in Nef-Hck-SH3 complex formation. Upon which, activation of Hck-mediated
kinase pathways aid Nef's viral replication and immune evasion, potentially via MHC-I degradation.
