---
title: "Characterising Disordered Region of Ire1 Luminal Domain"
layout: post
categories: Research
---

![HivProjectCover](/assets/img/research/ire1/cover.png){: width="70%" }

This study investigates key residues in the luminal domain of Ire1 (Ire1-LD) crucial for interaction with BiP and unfolded proteins, focusing on the conserved, highly-dynamic C-terminal region. Optimized conditions for small-scale Ire1-LD expression and purification enabled sufficient yield for biophysical characterisation, including oligomer formation analysis with BiP and ΔEspP. Emphasis is placed on avoiding freeze-thaw cycles to maintain sample integrity, enhancing the efficiency of future Ire1-LD studies.


Unfolded proteins in the Endoplasmic Reticulum (ER) trigger the Unfolded Protein Response (UPR) to restore protein folding balance. Ire1, a conserved UPR sensor, oligomerises upon detecting unfolded proteins or binding to the ER chaperone BiP. This study examines key Ire1 residues critical for interaction with BiP and unfolded proteins, focusing on the conserved but disordered C-terminal region of Ire1 luminal domain (Ire1-LD). Specific mutations were introduced at or in proximity to the oligomerisation interface. This study optimised conditions for small-scale Ire1-LD expression and purification which generated sufficient yield for further biophysical characterisation of Ire1-LD by analysing the formation of soluble Ire1-LD oligomers in the presence of BiP and a model substrate, ΔEspP. Information that can be acquired encompasses protein quality control, monitoring conformational changes, analysing molecular mass differences due to mutations, and detecting local chemical changes due to mutations. Furthermore, this study emphasises the importance of avoiding freeze-thaw cycles of Ire1-LD samples. Collectively, these methodologies enhance future Ire1-LD characterisation efficiency, permitting conducting the experiment in parallel settings.

## Optimisation of small-scale Ire1-LD expression and purification using IMAC

Each Ire1-LD plasmids of all mutants were designed with a TEV cleavable N-terminal 6x His tag. Incorporation of 6x His tags improves downstream protein purity and yield.  Once all purified Ire1-LD DNA variants were validated by DNA sequencing, each Ire1-LD DNA of all mutants were transformed into competent E. coli BL21 (DE3) cells, and IPTG-induced to allow their expression in a kanamycin-containing LB media, then analysed using SDS-PAGE.

![Figure1Expression](/assets/img/research/ire1/1.png){: width="100%" }
*Figure 1. Optimisation of small-scale expression of Ire1-LD variants.*
<div style="margin-top:-10px;"></div>
*All Ire1-LD variants were transformed in E. coli competent BL21(DE3) cells on LB broth containing kanamycin. All of the proteins were expressed using induction of IPTG at 1 mM final concentration. Induced proteins were lysed using BugBuster Master Mix. All samples including uninduced (protein samples prior to IPTG induction), induce whole cells (resuspended lysed induced samples after IPTG induction), and induced soluble samples (suspension of lysed induced samples after IPTG induction) were analysed. Uninduced samples (Un); Induced whole-cells samples (Ind w/c); Induced soluble samples (Ind Sol). Protein marker with molecular mass denoted in kDa (M). Ire1-LD with theoretical mass of 48 kDa, apparent mass at 55 kDa (highlighted in red box). Insoluble Ire1-LD proteins (highlighted with yellow arrows). Expression of soluble proteins (✔). Expression of insoluble protein (❌). 
(A) SDS-PAGE analysis of initial Ire1-LD expression leading to all soluble proteins except for two Ire1-LD mutants, BMB and BMAB. 
(B) SDS-PAGE analysis of optimised small-scale Ire1-LD expression for initially insoluble Ire1-LD mutants, BMB and BMAB.*

All Ire1-LD variants were produced as soluble proteins except for two Ire1-LD mutants. Their insolubility can be caused by several factors: (i) due to introducing mutations, or (ii) unfavourable experimental settings for their expression. To test my second hypothesis, I optimised the small-scale expression protocol by modifying the incubation expression settings, lowering the temperature for expression from 37°C to 18°C and increasing incubation period from 4 hours to overnight. In this revised approach, both mutants were expressed as soluble proteins. However, a major decrease in protein band intensity was observed. This suggests that expression of these mutants as soluble proteins were possible but remains limited. These convincing results highlight that mutation in these regions skewed the soluble nature of Ire1-LD to favour insolubility.

Once successfully expressed, these proteins were purified using a one step HisSpinTrap purification, yielding sufficient yield for downstream biophysical characterisation (approximately 1.2 mg per 100 ml culture).

## Ire1-LD oligomerises in the presence of ΔEspP

Characterising substrate-induced Ire1-LD oligomerisation requires utilising a model substrate for unfolded proteins. In this study, I used previously characterised ΔEspP resuspended and solubilised in DMSO (dimethyl sulfoxide) as the model peptide. ΔEspP has basic and hydrophobic amino acids which enables Ire1-LD binding in an amino-acid specific manner. Furthermore, ΔEspP has the highest binding affinity to Ire1-LD compared to other model substrates with 6.4 µM. This study focuses on analysing the development of soluble Ire1-LD oligomers within a concentration range of 0-15 µM ΔEspP. Higher concentration of ΔEspP used will induce insoluble Ire1-LD oligomers.

In this study, two approaches were utilised, Flow-induced dispersion analysis (FIDA) and mass photometry. In combination, these techniques permit obtaining experimental data across a wider range of concentrations, encompassing larger concentrations (FIDA) and smaller concentrations (mass photometry). FIDA provides valuable insights in protein quality control and conformational changes from aggregate formation while mass photometry extracts quantitative data on molecular mass changes, and the relative abundance of species in the sample analysed through molecular counting.

![Figure2Esp](/assets/img/research/ire1/2.png){: width="100%" }
*Figure 2. ΔEspP-induced Ire1-LD oligomerisation analysis using FIDA and mass photometry.*
<div style="margin-top:-10px;"></div>
*(A) FIDA analysis on 10 µM wild-type Ire1-LD in the absence and presence of ΔEspP. Ire-LD in the absence of ΔEspP (blue), Ire1-LD in the presence of 1 µM ΔEspP (red), Ire1-LD in the presence of 10 µM ΔEspP (yellow). (B) Mass photometry analysis of 0.2 µM wild-type Ire1-LD in the absence and presence of ΔEspP. Ire-LD in the absence of ΔEspP (dark blue), Ire1-LD in the presence of 1 µM ΔEspP (dark brown), Ire1-LD in the presence of 2 µM ΔEspP (dark green). The molecular mass in kDa (MW) and the number of counts are labelled. The dimeric (~96 kDa), tetrameric (~192 kDa), and higher order oligomers (>192 kDa) molecular weight of Ire1-LD are denoted with straight vertical red lines.*

Results from FIDA analysis shows the formation of multiple spikes in the addition of 10 µM ΔEspP which was not visible in the addition of 1 µM ΔEspP (Figure 2A). This suggests that ΔEspP-induced Ire1-LD oligomers formation correlates to the concentration of ΔEspP added. Further analysis using a more physiological concentration of Ire1-LD was able to highlight more in vitro changes upon ΔEspP addition. In the absence of ΔEspP, Ire1-LD exhibits its dimeric state at 115 kDa with 647 events. In the presence of 1 µM ΔEspP, the majority of Ire1-LD forms a higher-order oligomer which exceeds the capacity detected on a mass photometer, showing only a lesser populated Ire1-LD dimers at 91 kDa with 175 events. Doubling the addition of ΔEspP at 2 µM, recorded a distinct peak with molecular mass at 579 kDa and event counts at 728, amplifying ΔEspP-induced Ire1-LD oligomerisation. Both FIDA and mass photometry results were in agreement.

## BiP reverses ΔEspP-induced Ire1-LD oligomerisation

In addition to using ΔEspP, this study aimed to investigate the influence of adding BiP to ΔEspP-induced Ire1-LD oligomers. BiP interacts exclusively with Ire1-LD, not ΔEspP. Thus, competitive binding between BiP and ΔEspP for Ire1-LD does not exist. Hence, Ire1-LD oligomeric changes in the presence of BiP and ΔEspP were derived from direct BiP and Ire1-LD interactions.

![Figure3Bip](/assets/img/research/ire1/3.png){: width="70%" }
*Figure 3. BiP deoligomerises ΔEspP-induced Ire1-LD oligomers.*
<div style="margin-top:-10px;"></div>
*This figure highlights mass photometry analysis of Ire1-LD oligomerisation state from the addition of ATP-bound BiP. Ire1-LD in the absence of both BiP and ΔEspP (dark blue), Ire1-LD in the presence of 1 µM ΔEspP (dark brown), Ire1-LD in the presence of 2 µM ΔEspP (green),  Ire1-LD in the presence of ATP-bound BiP and 1 µM ΔEspP (purple). The dimeric (~96 kDa) and tetrameric (~192 kDa) molecular weights of Ire1-LD are denoted with straight vertical red lines. The molecular mass in kDa (MW) and the number of counts are labelled.*

For this section of the experiment, I used mass photometry to monitor molecular mass changes in the previously added 1 µM ΔEspP to Ire1-LD. Adding ATP-bound BiP induces deoligomerisation of the majority of Ire1-LD population to its dimeric state at 120 kDa with 609 counts. However, this Ire1-LD dimer was distinct from Ire1-LD in the absence of ΔEspP. This suggests that BiP was able to but not completely deoligomerise Ire1-LD oligomers.

## Freezing-thawing action reduces the structural integrity of the Ire1-LD samples

Due to the known risk of freezing purified protein samples. I tested whether frozen purified Ire1-LD samples can be used in downstream biophysical characterisation, exhibiting similar results to its fresh state. My results showed that freezing-thawing purified Ire1-LD samples led to the recovery of only less than 10% of protein yield (Figure 4A). Additionally, frozen Ire1-LD sample was found unresponsive to the addition of its ligand, ΔEspP (Figure 4B). Further comparison to its fresh state showed drastic differences in their molecular mass. Frozen Ire1-LD exhibited a higher order oligomer mass at 189 kDa while fresh Ire1-LD had 115 kDa. Collectively, these results suggest that freezing-thawing action of purified Ire1-LD samples led to an alteration in their structural integrity leading to its insensitivity to substrate-induced oligomerisation and the formation of aggregation.

![Figure4Frozen](/assets/img/research/ire1/4.png){: width="100%" }
*Figure 4. Characterisation of Frozen wild-type Ire1-LD protein.*
<div style="margin-top:-10px;"></div>
*(A) Fluorescence Induced Dispersion Analysis (FIDA) of 10 µM frozen wild-type Ire1-LD in the absence (blue) and presence of model substrate, 10 µM ΔEspP (red). (B) Mass photometry analysis between fresh (dark blue) and frozen (brown) wild-type Ire1-LD samples. The molecular mass in kDa (MW) and the number of counts are labelled above each peaks. The dimeric (~96 kDa) and tetrameric (~192 kDa) molecular weights of Ire1-LD are denoted with straight vertical red lines.*

## Biophysical characterisation of Ire1-LD using amide NMR

Next, this study investigated the impact of introducing mutations using amide NMR. Amide NMR is suitable to study the highly disordered but conserved region of the Ire1-LD oligomerisation interface. Due to its heterogeneity, a disordered region of a protein exhibits suitable property for NMR analysis. Folded regions of a protein are more compact and homogeneous, this leads to limitations in analysing data, rendering them ‘invisible’ in NMR spectra. To enhance NMR resolution, tested Ire1-LD samples involving wild-type and one Ire1-LD mutant, FML (predicted to have functional importance in Ire1-LD oligomerisation) were isotopically labelled with 15N.

Overlapped amide NMR spectra of both samples exhibit only the visible part of Ire1-LD - the disordered region (Figure 5). These signals represent each peak of Ire1-LD residues. Most peaks in mutant Ire1-LD have negligible differences compared to wild-type Ire1-LD. The most notable differences lie in the mutation sites introduced in this study (335TPST338 to GSSG). In this study, amide NMR was capable of highlighting perturbations in the chemical environment of each residue due to mutagenesis, deducing whether mutations had significant conformational impacts or local impacts. In this case, mutagenesis led to local changes instead of long-range changes.

![Figure5Nmr](/assets/img/research/ire1/5.png){: width="100%" }
*Figure 5. Overlapped 1H-15N HSQC NMR spectra of wild-type and mutant Ire1-LD.*
<div style="margin-top:-10px;"></div>
*This figure shows a zoomed-in amide NMR spectra highlighting mutated residues in FML Ire1-LD mutant. Annotated residues involved in mutagenesis are labelled. 
Wild-type Ire1-LD (red); FML Ire1-LD Mutant (black).*
